### PR TITLE
Implement helper methods

### PR DESCRIPTION
**Pull request by dkniffin**
I added a bunch of helpful methods for getting to the data quicker.

### CODE REVIEW

File: lib/ndbc/station.rb [hunk 1]
```diff
@@ -15,20 +15,74 @@ def standard_meteorological_data
...
+      else
+        super
+      end
+    end
+
     private
 
+    def latest_data(dataset)
+      send(dataset)[:values].sort_by do |row|
+        "#{row['YY']}#{row['MM']}#{row['DD']}"
```
Comment (ryanhertz) [inline lib/ndbc/station.rb]: Do you need to include the hours and minutes here too?

Comment (dkniffin) [inline lib/ndbc/station.rb]: Oh, yep. Good catch. I'll add those soon.

**Comment by dkniffin** [comment]
@ryanhertz Alright I added that change in. I've got some more changes coming, but they're unrelated, so I'll open another PR.

**Comment by ryanhertz** [comment]
Great, thanks! Let me know when you want me to push out a new version.

**Comment by dkniffin** [comment]
Thanks, will do!

**Pull request merged**